src/plugins/kbn_vislib_vis_types/public/tile_map.js

```diff
@@ -46,50 +72,35 @@ export default function TileMapVisType(Private, getAppState, courier, config) {
 
         pushFilter(filter, false, indexPatternName);
       },
-      mapMoveEnd: function (event) {
+      mapMoveEnd: function (event, uiState) {
+        const mapPrecision = zoomPrecision[event.zoom];
+        uiState.set('vis.params.mapCenter', event.center);
```

Comment:
Why vis.params? these can just be `'mapCenter``, `'mapZoom'`, and `'mapPrecision'` I think. Otherwise it's confusing (leads people to think these set the params on the vis somehow).